File: app/src/main/java/com/yatatsu/passiveviewsample/ui/user/UserScreen.java
package com.yatatsu.passiveviewsample.ui.user;

import com.yatatsu.passiveviewsample.ui.base.Screen;


public interface UserScreen extends Screen {
    void showUserImage(String avatarUrl);
    void showUserName(String fullName);
    void showErrorDialog(String message);
    void navigateToReposScreen(String username);
}

File: app/src/main/java/com/yatatsu/passiveviewsample/ui/user/UserScreen.kt
package com.yatatsu.passiveviewsample.ui.user

import com.yatatsu.passiveviewsample.ui.base.Screen


interface UserScreen : Screen {
  fun showUserImage(avatarUrl: String)
  fun showUserName(fullName: String)
  fun showErrorDialog(message: String)
  fun navigateToReposScreen(username: String)
}
